.agent/skills/deep-code-analyzer/scripts/scan_structure.py

```
@@ -0,0 +1,339 @@
+    print(f"🔍 Scanning: {project_root}")
+    print(f"   Max depth: {args.max_depth}")
+    print(f"   Include hidden: {args.include_hidden}")
+    print()
+
+    data = scan_directory(project_root, args.max_depth, args.include_hidden)
+
+    output_file = output_dir / "structure_data.json"
+    with open(output_file, "w", encoding="utf-8") as f:
+        json.dump(data, f, indent=2, ensure_ascii=False)
+
+    print(f"✅ Scan complete!")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)